libraries/radpdfprocessing/cross-platform.md

```diff
@@ -77,6 +81,63 @@ The new **FixedExtensibilityManager** class is exposing one property:
         FixedExtensibilityManager.FontsProvider = fontsProvider;
     {{endregion}}
 
+* **JpegImageConverter**: Gets or sets a *JpegImageConverterBase* instance used to provide a Jpeg converted image data.
```

Comment:
In addition to the framework limitations, describe why we have to convert the images